third_party/README.rst

```diff
@@ -7,11 +7,11 @@ This part of the repository contains extra dependencies required to build DALI,
 +==================================================================+===========================================================================================================================+===================================================================================================================+
 | `Google Benchmark <https://github.com/google/benchmark>`_        | `1.9.1 <https://github.com/google/benchmark/releases/tag/v1.9.1>`_                                                        | `Apache License 2.0 <https://github.com/google/benchmark/blob/master/LICENSE>`_                                   |
 +------------------------------------------------------------------+---------------------------------------------------------------------------------------------------------------------------+-------------------------------------------------------------------------------------------------------------------+
-| `Boost Preprocessor <https://github.com/boostorg/preprocessor>`_ | `1.86.0 <https://github.com/boostorg/preprocessor/releases/tag/boost-1.86.0>`_                                            | `Boost Software License 1.0 <https://github.com/boostorg/boost/blob/master/LICENSE_1_0.txt>`_                     |
+| `Boost Preprocessor <https://github.com/boostorg/preprocessor>`_ | `1.87.0 <https://github.com/boostorg/preprocessor/releases/tag/boost-1.87.0>`_                                            | `Boost Software License 1.0 <https://github.com/boostorg/boost/blob/master/LICENSE_1_0.txt>`_                     |
```

Comment:
Tag `1.87.0` points to the same commit as `1.86.0` (see https://github.com/boostorg/preprocessor/tags), so this is just a cosmetic change.